src/Build/Evaluation/Profiler/EvaluationLocationTabSeparatedPrettyPrinter.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-----------------------------------------------------------------------
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Microsoft.Build.Framework.Profiler;
+
+namespace Microsoft.Build.Evaluation
+{
+    /// <summary>
+    /// Pretty prints an evaluation location in tab separated value (TSV) format
+    /// </summary>
+    internal sealed class EvaluationLocationTabSeparatedPrettyPrinter : EvaluationLocationPrettyPrinterBase
+    {
+        /// <inheritdoc/> 
+        internal override void AppendHeader(StringBuilder stringBuilder)
+        {
+            stringBuilder.AppendLine("Id\tParentId\tPass\tFile\tLine #\tExpression\tInc (ms)\tInc (%)\tExc (ms)\tExc (%)\t#\tKind\tBug");
```

Comment:
Nit: pull the names up in the superclass

Reply:
I'll lump this together with the work associated to the comment below

---
In reply to: [159566340](https://github.com/Microsoft/msbuild/pull/2837#discussion_r159566340) [](ancestors = 159566340)